.github/workflows/oclif-release.yml

```diff
@@ -25,8 +25,8 @@ jobs:
       matrix:
         include:
           - os: macos-latest
-            asset_name: twilio-v${{ github.event.inputs.tag-name }}
-            command_name: npx oclif-dev pack --xz -t darwin-arm64,darwin-x64,linux-arm,linux-x64,win32-x64,win32-x86
```

Comment:
Do we need to remove the build targets too? IIRC`twilio update` refers the vanilla builds from s3. How does it work after this change?

Reply:
Yes, I missed that. Thanks!!